.github/workflows/on_pr_master.yml

```diff
@@ -45,9 +67,10 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
   build:
+    needs: changes
     # Macos includes virtualbox and vagrant.
     runs-on: macos-latest
-
+    if: ( ( github.event.action == 'closed' && github.event.pull_request.merged == true ) && contains(github.event.pull_request.labels.*.name, 'major')||contains(github.event.pull_request.labels.*.name, 'minor')||contains(github.event.pull_request.labels.*.name, 'patch') ) || ( github.event.action != 'closed' && needs.changes.outputs.not-doc == 'true' )
```

Comment:
Is it possible to extract and store the value of this 
```
github.event.pull_request.labels.*.name
```
in a variable and use it wherever required?

Reply:
Not as I am aware of.  Maybe you could have a look?

Reply:
I am able to extract and store the value in an env variable but i get a syntax error when i use it inside ```contains```. I think we can merge this PR for now as it would do the work and can modify  the code once i get hold of the right way to use the variable in ```contains```.